CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+### Added
+
+- Added a way to turn of redirecting output to stderr and stdout
```

Comment:
I think this needs to be updated, and would probably be better to put under the `## Changed` section